test_common/harness/testHarness.h

```diff
@@ -28,8 +28,8 @@ class Version {
     Version(): m_major(0), m_minor(0) {}
 
     Version(cl_uint major, cl_uint minor): m_major(major), m_minor(minor) {}
-    int major() const { return m_major; }
-    int minor() const { return m_minor; }
+    int version_major() const { return m_major; }
+    int version_minor() const { return m_minor; }
```

Comment:
Nit, but would probably go for `get_major()`/`get_minor()` since `version` feels redundant as it's implied already on the class.